uber/tests/models/test_attendee.py

```diff
@@ -90,6 +90,18 @@ def test_last_first(monkeypatch):
     assert 'xxx' == Attendee(first_name='x', last_name='y').last_first
 
 
+def test_legal_name_same_as_full_name():
+    same_legal_name = Attendee(first_name='First', last_name='Last', legal_name='First Last')
+    same_legal_name._misc_adjustments()
+    assert '' == same_legal_name.legal_name
+
+
+def test_legal_name_diff_from_full_name():
+    diff_legal_name = Attendee(first_name='first', last_name='last', legal_name='diff name')
+    diff_legal_name._misc_adjustments()
+    assert 'diff name' == diff_legal_name.legal_name
```

Comment:
The interesting thing about this (as we discovered discussing this on slack) is that even if you used
```python
diff_legal_name = Attendee(first_name='diff', last_name='name', legal_name='diff name')
```
this test would still pass because the code automatically title cases `first_name` and `last_name`.

No changes requested, just thought that was interesting!